codespell_lib/data/dictionary.txt

```diff
@@ -10556,8 +10556,16 @@ prepaired->prepared
 prepartion->preparation
 prepartions->preparations
 prepatory->preparatory
+prependet->prepended
+prepent->prepend, disabled due being a programming term
+prepented->prepended
 preperation->preparation
 preperations->preparations
+preppend->prepend
+preppended->prepended
+preppendet->prepended
+preppent->prepend
```

Comment:
This should probably suggest preprent then, although I don't know what Travis will think of that.

Reply:
What about using both, preprent and prepend like:

``
preppent->prepend, preprent,
``

Reply:
Yes sorry, I think that's what I meant. I think someone needs to decide if it having lots of use as a programming term means its a valid spelling, in which case we should treat it as a valid word and suggest it here as the closest spelling. Or if it's a typo primarily, we should suggest prepend as the closest valid word (although curiously Firefox dictionary doesn't like it!).

Reply:
I have never seen `prepent`, where does it get used?

Reply:
Oh, I see one example above. I'm not sure it's prevalent enough to justify the misses it'll cause, I'd rather make people whitelist it. If it's very common in some language I'm less familiar with (of which there are many -- mostly I'm Python/C/C++!) then I'm okay allowing it. FF dictionary not allowing also makes me think more that we shouldn't.

Reply:
I'm currently also unsure which way to go so i have split the other spelling corrections into a separate PR https://github.com/codespell-project/codespell/pull/863